selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/NavigationToolbar.java

```diff
@@ -546,26 +546,22 @@ public void goToAdminNotifications() {
    */
   public void goToAddUser() {
     info("Go to add user page");
-    int repeat = 0;
-    while (evt.waitForAndGetElement(ELEMENT_ADMINISTRATION_USERS, 3000, 0) == null) {
-      if (repeat > 5)
-        break;
-      info("Click on administration icon");
-      evt.click(ELEMENT_TOOLBAR_ADMINISTRATION);
-      repeat++;
-    }
-    info("add user page is shown");
-    info("evt.mouse over on community link");
-    evt.mouseOver(ELEMENT_ADMINISTRATION_USERS, true);
-    if (evt.waitForAndGetElement(ELEMENT_ADMINISTRATION_PORTAL_ADD_USERS, 3000, 0) != null) {
-      info("evt.click on Add user link");
-      evt.click(ELEMENT_ADMINISTRATION_PORTAL_ADD_USERS);
-    } else {
-      info("Cannot evt.click on add user link. Go to this page by link");
-      testBase.getExoWebDriver().getWebDriver().get(testBase.getExoWebDriver().getBaseUrl() + "/g/:platform:administrators/administration/newStaff");
-    }
+
+    info("Click on administration icon");
+    $(ELEMENT_TOOLBAR_ADMINISTRATION).waitUntil(Condition.appears,10000);
+    $(ELEMENT_TOOLBAR_ADMINISTRATION).click();
+    $(byText("Community")).waitUntil(Condition.appears,10000);
+    $(byText("Community")).hover();
+    $(byText("Add Users")).click();
   }
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.